.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: python
 python:
-  - "3.8"
-# command to install dependencies
+- '3.8'
 install:
-  - pip install -r requirements.txt
-# command to run tests
+- pip install -r requirements.txt
 script:
-  - pip install flask_api
-  - pip install flask-sqlalchemy psycopg2 flask-migrate
-  - pytest
+- pip install flask_api
+- pip install flask-sqlalchemy psycopg2 flask-migrate
+- pytest
```

Comment:
We should try to get all of our tests inside a `test` folder and see if pytest will still